### PR TITLE
Minor cleanup / update gulp

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Updated gulp to `3.9.1` from `3.9.0`.
- Removed extraneous whitespace from dependencies.
- Minor normalizing of spacing in parentheses.
- Fixed max line length warnings in linter.

## Review

- @contolini 
- @Scotchester 

## Notes

- https://github.com/cfpb/front-end/pull/111 needs to go in in order for the linter to pass.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -166,18 +166,18 @@ var CapitalFrameworkGenerator = yeoman.generators.Base.extend({
         message:  'Which build tool do you prefer?',
         choices:  [
           {
-            name: 'Grunt',
+            name:  'Grunt',
```
Comment (jimmynotjim) [inline app/index.js]: Are we continuing to support Grunt?

**Comment by jimmynotjim** [comment]
👍

File: app/index.js (hunk 1 above)
Comment (Scotchester) [inline app/index.js]: I vote yes, until the level of effort to continue supporting it gets too high.

Comment (anselmbradford) [inline app/index.js]: @contolini @Scotchester

**Comment by Scotchester** [comment]
:+1:

**Pull request merged**